src/contexts/PokemonDataContext.js

```diff
@@ -7,96 +8,197 @@ export const PokemonDataContext = createContext();
 export const PokemonDataProvider = ({ children }) => {
   const [allPokemons, setAllPokemons] = useState();
   const [pokemons, setPokemons] = useState();
+  const [favouritesPokemons, setFavouritesPokemons] = useState();
```

Comment:
I'm pretty sure that it's a typo - it should be "favouritePokemons" (without first "s")